packages.yaml

```diff
@@ -172,6 +172,7 @@ packages:
       - tcl@8.6.6
       - tk@8.6.6 ^tcl@8.6.6
       - mercurial@4.1.2 ^python@2.7.13
+      - bowtie2@2.3.1
```

Comment:
Running:
```console
$ spack spec bowtie2@2.3.1
Input spec
--------------------------------
bowtie2@2.3.1

Normalized
--------------------------------
bowtie2@2.3.1
    ^readline
        ^ncurses
            ^pkg-config
    ^tbb
    ^zlib

Concretized
--------------------------------
bowtie2@2.3.1%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
    ^readline@7.0%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
        ^ncurses@6.0%gcc@4.8.5~symlinks arch=linux-rhel7-x86_E5v4_Mellanox 
            ^pkg-config@0.29.2%gcc@4.8.5+internal_glib arch=linux-rhel7-x86_E5v4_Mellanox 
    ^tbb@2017.5%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
    ^zlib@1.2.11%gcc@4.8.5+pic+shared arch=linux-rhel7-x86_E5v4_Mellanox 
```
I see that `bowtie2` depends on `tbb`. We need to configure `intel%tbb` in `packages.yaml` before merging this, so that `bowtie2%intel` uses the binaries that come with the external Intel installation.